offline_signing/transfer.sh

```diff
@@ -49,6 +49,19 @@ upload() {
         --board="${board}" \
         --version="${version}" \
         --payload="${payload}"
+
+    # Update version in a canary channel if one is defined.
+    local channel_var="ROLLER_CANARY_CHANNEL_$(tr a-z A-Z <<<${channel})"
```

Comment:
Minor nit, but maybe helps readability?  This would allow using it as a regular variable without the `!` indirection:

    local -n canary_channel="ROLLER_CANARY_CHANNEL_${channel^^}"

Reply:
Two tricks I didn't know about.  Will fix.